### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Pulls changes from PR 1135 which was left around stale.

Authored by: jtcarnes

### CODE REVIEW

**Review by niladrih — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -125,7 +137,8 @@ buf.as_slice().into_iter().map(|b| assert_eq!(b, 0xff)).for_each(drop);
 We think this can help a lot of database projects as well, where they typically have all the smarts in their database engine
 and they want the most simple (but fast) storage device. For a more elaborate example see some of the tests in mayastor/tests.
 
-To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local storage and remote storage using NVMe-oF TCP. Another advantage of the implementation is that if you were to remove
+To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local
+storage and remote storage using NVMe-oF TCP. Another advantage of the implementation is that if you were to remove
```
Comment (avishnu) [inline README.md:141]: Why is there a newline break?

File: README.md [hunk 2]
```diff
@@ -125,7 +137,8 @@ buf.as_slice().into_iter().map(|b| assert_eq!(b, 0xff)).for_each(drop);
 We think this can help a lot of database projects as well, where they typically have all the smarts in their database engine
 and they want the most simple (but fast) storage device. For a more elaborate example see some of the tests in mayastor/tests.
 
-To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local storage and remote storage using NVMe-oF TCP. Another advantage of the implementation is that if you were to remove
+To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local
```
Comment (avishnu) [inline README.md:140]: Why is there a newline break?

Comment (tiagolobocastro) [reply README.md:140]: Just to keep each line short in "code". Once rendered there's no line break

**Review by avishnu — APPROVED** [review]

**Review by dsavitskiy — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors cancel

**Comment by tiagolobocastro** [comment]
nvm, I got confused..
bors merge

**Pull request merged**